Fix aclocal warning:
/usr/share/aclocal/ORBit.m4:4: warning: underquoted definition of AM_PATH_ORBIT
--- ORBit-0.5.17/ORBit.m4
+++ ORBit-0.5.17/ORBit.m4
@@ -3,3 +3,3 @@
 dnl
-AC_DEFUN(AM_PATH_ORBIT,
+AC_DEFUN([AM_PATH_ORBIT],
 [dnl 
